socorro/external/rabbitmq/crashstorage.py

```diff
@@ -151,7 +151,7 @@ def _basic_get_transaction(self, conn, queue):
         """reorganize the the call to rabbitmq basic_get so that it can be
         used by the transaction retry wrapper."""
         things = conn.channel.basic_get(queue=queue)
-        return things
+        return conn.channel, things
```

Comment:
I think it'd be better to unwrap `things` and return a non-nested datastructure. That'd improve readability of this code. I think I would put it all in a dict like this:

```
things = conn.channel.basic_get(queue=queue)
return {
    'channel': conn.channel,
    'method_frame': things[0],
    'header_frame': things[1],
    'body': things[2]
}
```

That affects the code below, too.